src/main/java/com/iridium/iridiumskyblock/utils/LocationUtils.java

```diff
@@ -82,4 +84,19 @@ private static Location getHighestLocation(int x, int z, World world) {
         return block.getLocation().add(0.5, 0, 0.5);
     }
 
+    /**
+     * With the data pack, you can modify the height limits and in the Spigot API.
+     * It exists since 1.17 on Spigot and 1.16 at PaperMC.
+     *
+     * @param world The world
+     * @return The lowest AIR location.
+     */
+    public static int getMinHeight(World world) {
+        if (getVersion() >= 17) {
+            return world.getMinHeight(); // Available only in 1.17 Spigot and 1.16.5 PaperMC
+        } else {
+            return 0;
+        }
```

Comment:
Simpfly it with conditional operators (?:) its better :panda:

Reply:
effectively :)